api/src/main/java/org/pmiops/workbench/api/AuthDomainController.java

```diff
@@ -44,22 +28,4 @@ public ResponseEntity<AuthDomainCreatedResponse> createAuthDomain(String authDom
             .authDomainName(authDomainName)
             .groupEmail(group.getGroupEmail()));
   }
-
-  @Deprecated // use UserAdminController.updateAccountProperties()
-  @Override
-  @AuthorityRequired({Authority.ACCESS_CONTROL_ADMIN})
-  public ResponseEntity<Void> updateUserDisabledStatus(UpdateUserDisabledRequest request) {
-    final DbUser targetDbUser = userService.getByUsernameOrThrow(request.getEmail());
-    final Boolean previousDisabled = targetDbUser.getDisabled();
-    final DbUser updatedTargetUser =
-        userService.setDisabledStatus(targetDbUser.getUserId(), request.getDisabled());
-    auditAdminActions(request, previousDisabled, updatedTargetUser);
-    return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
-  }
-
-  private void auditAdminActions(
-      UpdateUserDisabledRequest request, Boolean previousDisabled, DbUser updatedTargetUser) {
-    authDomainAuditAdapter.fireSetAccountDisabledStatus(
```

Comment:
this was the only remaining instance of AuthDomainAuditAdapter, so it can be removed.

Equivalent functionality is now handled by `profileAuditor.fireUpdateAction()`